<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>11</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>11</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>62</y>
     <width>1284</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Step 1 - Calculate Mask</string>
    </attribute>
    <widget class="QComboBox" name="USMaskkHzDropDown">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>55</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>60</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>US Frequency (kHz)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>95</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PPW</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="USPPWSpinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>92</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>6.000000000000000</double>
     </property>
     <property name="maximum">
      <double>12.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>6.000000000000000</double>
     </property>
    </widget>
    <widget class="QWidget" name="USMask" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>1013</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CalculatePlanningMask">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>352</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Calculate planning
mask</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="CTZTETabs">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>173</y>
       <width>219</width>
       <height>145</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ZTEtab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>ZTE</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>59</y>
         <width>193</width>
         <height>40</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="LayRange"/>
      </widget>
      <widget class="QLabel" name="RangeLabel">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>24</y>
         <width>165</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Normalized ZTE Range</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>CT</string>
      </attribute>
      <widget class="QDoubleSpinBox" name="HUThresholdSpinBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>46</y>
         <width>97</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>300.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>23</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>HU threshold</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QScrollBar" name="TransparencyScrollBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>974</x>
       <y>452</y>
       <width>194</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>451</y>
       <width>115</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>T1W transparency</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="HideMarkscheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>448</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide marks</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="outputTerminal">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>607</y>
     <width>1282</width>
     <height>143</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>114</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Terminal output</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>12</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LIFU System</string>
   </property>
  </widget>
  <widget class="QLabel" name="TXLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>11</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>11</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Thermal profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="ThermalProfileLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>206</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>USMaskkHzDropDown</tabstop>
  <tabstop>USPPWSpinBox</tabstop>
  <tabstop>CTZTETabs</tabstop>
  <tabstop>CalculatePlanningMask</tabstop>
  <tabstop>HideMarkscheckBox</tabstop>
  <tabstop>outputTerminal</tabstop>
  <tabstop>HUThresholdSpinBox</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
